pipeline/tasks/protoc_tasks.py

```diff
@@ -218,8 +259,8 @@ def execute(self, src_proto_path, import_proto_path, output_dir,
         #   - it doesn't have to
         #   - and multiple invocation will overwrite the desc_out_file
         self.exec_command(
-            _protoc_header_params(
-                header_proto_path, toolkit_path) +
+            ['protoc'] +
```

Comment:
I think using `proto_params.proto_compiler` is fine here as well and I believe that's more consistent. You need to change a baseline for ruby_gapic_client_pipeline.